scripts/test/Muon/muon_context_test.py

```diff
@@ -0,0 +1,118 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+import sys
+import unittest
+
+from mantid.api import AnalysisDataService
+from mantid.api import FileFinder
+from mantid.dataobjects import Workspace2D
+
+from Muon.GUI.Common.contexts.muon_context import MuonContext
+from Muon.GUI.Common.contexts.muon_data_context import MuonDataContext
+from Muon.GUI.Common.contexts.muon_group_pair_context import MuonGroupPairContext
+from Muon.GUI.Common.contexts.muon_gui_context import MuonGuiContext
+from Muon.GUI.Common.muon_load_data import MuonLoadData
+from Muon.GUI.Common.utilities.load_utils import load_workspace_from_filename
+
+if sys.version_info.major < 2:
```

Comment:
I don't want to hold up merging this as it's good set of changes but I think these lines are redundant?